### DISCUSSION TITLE
See renovate labels in Dependency Dashboard

### DISCUSSION THREAD

**Discussion by pbabbicola** [renovatebot/renovate#32404] [Suggest an Idea]
### Tell us more.

Scenario from: #32397
I would like to see renovate-assigned labels in my dependency dashboard.

Eg:

 - [ ] Update module google.golang.org/grpc to v1.67.1 `team1` `team2`

I'm not really picky at how they are shown and I'm not really good at making pretty things, so any way they are shown works for me!

**Comment by rarkins** [comment]
Using backticks for "code" for labels might be the best way, however we already have a similar visual look for listing package names when a branch contains more than one upgrade:

![image](https://github.com/user-attachments/assets/b68434e5-c111-4c56-9b43-729b3d9b3971)


We could _potentially_ use badges instead of `code` but I'm not sure if they'll distort the list spacing too much

**Comment by pbabbicola** [comment]
How can we help implement this? Do you have a decision on what you want it to look like?

**Reply by rarkins** [reply]
You can experiment (edit/preview/etc) with an existing dashboard to see if image badges can work without distorting the list spacing. It's important that it looks ok when it's combined with multi-package PRs such as in the screenshot above

**Reply by pbabbicola** [reply]
it's... not the prettiest: pbabbicola/test-renovate#1

**Comment by pbabbicola** [comment]
Hello @rarkins, I am wondering if perhaps an alternative solution for us would be to have a mergeable field for the commit message? Do you think that would be something that could be implemented?

For example, I've seen this in other parts of the stack (sorry for weird censoring, I have no idea what I can share and what I can't):
![Screenshot from 2025-04-01 19-33-32](https://github.com/user-attachments/assets/c52aa863-afde-4c0d-9c55-c3b6237f7b19)
This feels like it could be a good workaround if it were appendable?

**Reply by rarkins** [reply]
I think the dashboard lines are taken from prTitle, not commitMessage. These are typically the same though. If you took this approach then you'd need the PR title to include the labels, which is probably not a good idea.

**Reply by pbabbicola** [reply]
Yes, but they are typically the same, that is why I was thinking something like that?
But it could be for us an acceptable way for example, maybe not the best of course!

**Comment by rarkins** [comment]
Work to make dashboard content more customizable will proceed after https://github.com/renovatebot/renovate/issues/35663 is done

**Reply by pbabbicola** [reply]
Thank you so much <3.

**Reply by pbabbicola** [reply]
Please do let me know if there is anything to help. I am not very good at typescript but I can try to help, or test, or anything!
